src/environments/constants.ts

```diff
@@ -1,4 +1,5 @@
 export const constants = {
   trxRetry: 60, // 1 minute
-  userIdle: 900, // 15 mins in seconds
+  idleTime: 900,
+  idleTimeout: 300,
```

Comment:
What is the different between `idleTime` and `idleTimeout`? I was looking as the document https://github.com/rednez/angular-user-idle but they don't really explain it

Reply:
I was looking on this: https://github.com/rednez/angular-user-idle#service-logic
So idle time is how long user can be inactive before starting timeout.